src/EnergyPlus/TARCOGArgs.cc

```diff
@@ -425,10 +425,15 @@ int ArgCheck(EnergyPlusData &state,
             return ArgCheck;
         }
 
-        if (((LayerType(i)) != TARCOGLayerType::SPECULAR) && ((LayerType(i)) != TARCOGLayerType::WOVSHADE) &&
-            ((LayerType(i)) != TARCOGLayerType::VENETBLIND_HORIZ) && ((LayerType(i)) != TARCOGLayerType::PERFORATED) &&
-            ((LayerType(i)) != TARCOGLayerType::DIFFSHADE) && ((LayerType(i)) != TARCOGLayerType::BSDF) &&
-            ((LayerType(i)) != TARCOGLayerType::VENETBLIND_VERT)) {
+        if BITF_NOT_TEST_ANY(BITF(LayerType(i)), BITF(TARCOGLayerType::SPECULAR) |
+                                                   BITF(TARCOGLayerType::WOVSHADE) |
+                                                   BITF(TARCOGLayerType::VENETBLIND_HORIZ) |
+                                                   BITF(TARCOGLayerType::PERFORATED) |
+                                                   BITF(TARCOGLayerType::DIFFSHADE) |
+                                                   BITF(TARCOGLayerType::BSDF) |
+                                                   BITF(TARCOGLayerType::VENETBLIND_VERT))
+
+        {
             ArgCheck = 22;
```

Comment:
So, `BITF_NOT_TEST_ANY` is a bad name, a better one would be `BITF_TEST_NONE` (i.e., it matches none of the values). Alternatively you can also just use `!BITF_TEST_ANY`.

Reply:
Makes sense! Thank you.